README.md

```diff
@@ -148,23 +148,21 @@ If you are trying to use the P2 plugin permanently, it likely makes more sense t
    ```
    And
    ```
-   + <feature name="nexus-repository-p2" description="org.sonatype.nexus.plugins:nexus-repository-p2" version="0.0.2">
+   + <feature name="nexus-repository-p2" description="org.sonatype.nexus.plugins:nexus-repository-p2" version="1.0.0">
    +     <details>org.sonatype.nexus.plugins:nexus-repository-p2</details>
-   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-p2/0.0.2</bundle>
+   +     <bundle>mvn:org.sonatype.nexus.plugins/nexus-repository-p2/1.0.0</bundle>
    + </feature>
     </features>
    ```
 This will cause the plugin to be loaded and started with each startup of Nexus Repository.
 
 ## The Fine Print
-
-It is worth noting that this is **NOT SUPPORTED** by Sonatype, and is a contribution of ours
-to the open source community (read: you!)
+Starting from version 3.21+ the `p2` plugin is supported by Sonatype, but still is a contribution of ours to the open source community (read: you!)
 
 Remember:
 
 * Use this contribution at the risk tolerance that you have
```

Comment:
I think this line can be struck.

Reply:
@jlstephens89 Is this still needed?

Reply:
Done 018a1dddbd5a3ffd0dc188c53c6f337b1e861677